OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -301,9 +301,9 @@ protected function generateNonce()
     /**
      * Configure the option resolver
      *
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    protected function configureOptions(OptionsResolverInterface $resolver)
+    protected function configureOptions(OptionsResolver $resolver)
```

Comment:
> Regarding the removal, there is no need to create a new branch with it removed yet. BC layers don't hurt much. It is not worth maintaining multiple parallel versions of the bundle just because of a BC break removing a small BC layer.

@stof Not sure of what you said. So you mean to keep BC by madding a news method and deprecated the old one without any removal version announcement?